seismicpro/src/refractor_velocity/refractor_velocity_field.py

```diff
@@ -141,6 +145,45 @@ def __str__(self):
 
         return msg
 
+    @classmethod
+    def from_file(cls, path, survey=None, is_geographic=None, auto_create_interpolator=True, encoding="UTF-8"):
+        """Load field with velocity models from a file.
```

Comment:
You refer almost everywhere as `near-surface velocity`
```suggestion
        """Load field with near-surface velocity models from a file.
```